docs/analyzer/user_guide.md

```diff
@@ -2579,3 +2593,30 @@ void testError2() {
   int x = 1 / 0;
 }
 ```
+
+### Setting with config file
+
+Review status can be configured by a config file in YAML format. This config
+file has to represent a list of review status settings:
+
+```yaml
+- filepath: /path/to/project/test/**
+  checker: core.DivideZero
+  message: Division by zero in test files is automatically intentional.
+  review_status: intentional
+- filepath: /path/to/project/important/module/**
+  message: All reports in this module should be investigated.
+  review_status: confirmed
+```
+
+The fields of a review status settings are:
+
+- `filepath` (optional): A glob to a path where the given review status is
```

Comment:
Should these be absolute filepaths?
If yes, would this file be transferrable between designers?

Should we use here `--trim-path-prefix <project>` root also for the analyze command?
or we recommend the users to use * in the prefix of the paths?
eg:
```*/test_lib/src/* ```

Reply:
Yes, the filter has to match the absolute path, even if `--trim-path-prefix` is used. I added this information to the documentation.